apps/total-typescript/studio/schemas/objects/resources/solution.js

```diff
@@ -30,7 +30,15 @@ export default {
       name: 'resources',
       title: 'Resources',
       type: 'array',
-      of: [{type: 'muxVideo'}, {type: 'stackblitz'}],
+      of: [
+        {
+          title: 'Video Resource',
+          type: 'reference',
+          to: [{type: 'videoResource'}],
```

Comment:
went through and updated all of the solutions and exercises that were already publish to have a videoResource instead of muxVideo